# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/encoder/MediaEncoder.cpp
# Commit: d0e8d413cd1c
# Full Hash: d0e8d413cd1c6aa414cf167b085f635ae857e408
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-10-02 16:35:54
# Regressor Bug: 1493613
# File Overlap Count: 1
# Description:
#   Bug 1454998 - Remove track blocking from MediaInputPort. r=karlt
#   
#   This functionality is not used since we moved to only having a single track per
#   MediaStream (bug 1493613).
#   
# ==============================================================================

diff -r 3ce4dc7e9ae2 -r d0e8d413cd1c dom/media/encoder/MediaEncoder.cpp
--- a/dom/media/encoder/MediaEncoder.cpp	Wed Oct 02 08:12:18 2019 +0000
+++ b/dom/media/encoder/MediaEncoder.cpp	Wed Oct 02 08:12:28 2019 +0000
@@ -522,8 +522,8 @@
     mPipeStream = AudioNodeStream::Create(ctx, engine, flags, ctx->Graph());
     AudioNodeStream* ns = aNode->GetStream();
     if (ns) {
-      mInputPort = mPipeStream->AllocateInputPort(aNode->GetStream(), TRACK_ANY,
-                                                  TRACK_ANY, 0, aOutput);
+      mInputPort =
+          mPipeStream->AllocateInputPort(aNode->GetStream(), 0, aOutput);
     }
   }
 